.github/workflows/java-cd.yml

```diff
@@ -99,15 +99,7 @@ jobs:
             - name: Set the release version
               shell: bash
               run: |
-                  if ${{ github.event_name == 'pull_request' || github.event_name == 'push' }}; then
-                    R_VERSION="255.255.255"
-                  elif ${{ github.event_name == 'workflow_dispatch' }}; then
-                    R_VERSION="${{ env.INPUT_VERSION }}"
-                  else
-                    R_VERSION=${GITHUB_REF:11}
-                  fi
-                  echo "RELEASE_VERSION=${R_VERSION}" >> $GITHUB_ENV
-                  echo "Release version detected: $R_VERSION"
```

Comment:
don't hardcode the version, we might need to push patches later on. The current code allows you to do one of:
1. push relevant tag v1.0.0
2. run the workflow manually with workflow_dispatch and pass the specific version
3. pull requests will be triggered with 255.255.255

Reply:
Reverted